README.md

```diff
@@ -24,7 +24,13 @@ We're working on [tutorials](https://napari.org/tutorials/), but you can also qu
 **napari** can be installed on most macOS, Linux, and Windows systems with Python 3.6, 3.7 and 3.8 by calling
 
 ```sh
-$ pip install napari
+$ pip install napari[pyside2]
```

Comment:
I think this needs a bit of extra narrative. Here's a proposed version:

"""
napari needs a library called Qt to run its user interface (UI). In Python, there are two alternative libraries to run this, called PyQt5 and PySide2. By default, we don't choose for you, and `pip install napari` will not install either. You might already have one of them installed thanks to other scientific packages such as Spyder and matplotlib. Once installed, running `napari` will result in an error message if neither is available.

To install napari *with* a UI framework, you can use `pip install napari[pyqt5]`, `pip install napari[pyside2]`, or `pip install napari[all]`. This last option will choose a framework for you — currently PyQt5, but this could change in the future.
"""

What do you think?